index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First project in Microverse</title>
+
+    <!-- custom css file link -->
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>
+    <h1 class="hello">Hello Microverse!</h1>
+</body>
```

Comment:
- [Optional] Please write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. This will make your repo more professional.
- [Optional] Kindly make your PR description to be more descriptive. The description should be a summary of all the changes made in the pull request. This will make your repo more professional. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information about professional repo.